cql3/authorized_prepared_statements_cache.hh

```diff
@@ -108,6 +109,10 @@ public:
         static void inc_evictions() noexcept {
             ++shard_stats().authorized_prepared_statements_cache_evictions;
         }
+
+        static void inc_new_gen_on_cache_size_eviction() noexcept {
+            ++shard_stats().authorized_prepared_statements_new_gen_on_cache_size_evictions;
```

Comment:
Is this a statistic that anybody will be able to understand? :-(

Reply:
Would putting an inline comment with an explanation make it better?

Reply:
No, the line of code is fine - it refers to a stat that you define (and explain) elsewhere, so no need for an extra comment. But the thing is that this whole statistic - "new gen on cache size evictions" is something that no mortal user will understand. 

Here's an idea: For the prepared statement cache, you **know** that the "new gen" will only be for used-once prepared statements. So you can use this phrase: "evicted prepared statement after having been used only once" (or some other variant of this phrase) - "used once" is much more self-explanatory than "new gen".

Reply:
Makes sense. I'll do something like you've suggested.